src/webnn/native/dml/ContextDML.h

```diff
@@ -34,6 +34,11 @@ namespace webnn::native::dml {
 
       private:
         GraphBase* CreateGraphImpl() override;
+        MaybeError ComputeImpl(GraphBase* graph,
```

Comment:
Is same for each backend? If so, how about moving this funtion into ContextBase?

Reply:
Yes, that's a good idea.